packages/react-component-library/src/components/Popover/Popover.tsx

```diff
@@ -80,28 +125,13 @@ export const Popover: React.FC<PopoverProps> = ({
    */
   return (
     <>
-      {/*
-      // @ts-ignore */}
+      {/* @ts-ignore */}
```

Comment:
Does this create a type error now? 

Previously the comment had to be in that weird format for the `@ts-ignore` to be respected.